pkg/process/metadata/workloadmeta/grpc.go

```diff
@@ -113,6 +115,15 @@ func (l *GRPCServer) StreamEntities(_ *pbgo.ProcessStreamEntitiesRequest, out pb
 				continue
 			}
 
+			// The diff received from the channel should be 1 + the previous version. Otherwise, we have lost data,
+			// and we should signal the client to resync by closing the stream.
+			log.Trace("[WorkloadMeta GRPCServer] expected diff version %v, actual %v", expectedVersion, diff.cacheVersion)
```

Comment:
These are ints right, so they should be `%d`
```suggestion
			log.Trace("[WorkloadMeta GRPCServer] expected diff version %d, actual %d", expectedVersion, diff.cacheVersion)
```

Reply:
`%v` should actually work as well. From the `fmt` docs https://pkg.go.dev/fmt#:~:text=%25v%09the%20value%20in%20a%20default%20format